java/Mekhmet-Efe_Adzhar/src/com/zoolatech/lecture6/tasks/_2/OrderCalc.java

```diff
@@ -0,0 +1,12 @@
+package com.zoolatech.lecture6.tasks._2;
+
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+interface OrderCalc {
```

Comment:
same: interfaces define a specific behaviour which classes need to implement; this looks more like a class since it already has the implementation